### PR TITLE
connecting peak -> coach and coach -> athlete

### PR DESCRIPTION
**Pull request by isabella-feng**
might need refinement - merging now to avoid more conflicts in the future

### CODE REVIEW

File: website/templates/coach_dashboard.html [hunk 1]
```diff
@@ -710,24 +658,24 @@ <h6 class="mb-1">Notes</h6>
     </div>
     <!-- container-scroller -->
     <!-- plugins:js -->
-    <script src="{{url_for('static', filename = 'assets/vendors/js/vendor.bundle.base.js')}}"></script>
```
Comment (tymkiwdylan) [inline website/templates/coach_dashboard.html:713]: Same thing

File: website/templates/coach_dashboard.html [hunk 2]
```diff
@@ -710,24 +658,24 @@ <h6 class="mb-1">Notes</h6>
...
+    <script src="assets/vendors/js/vendor.bundle.base.js"></script>
     <!-- endinject -->
     <!-- Plugin js for this page -->
-    <script src="{{url_for('static', filename='assets/vendors/chart.js/Chart.min.js')}}"></script>
-    <script src="{{url_for('static', filename='assets/vendors/progressbar.js/progressbar.min.js')}}"></script>
```
Comment (tymkiwdylan) [inline website/templates/coach_dashboard.html:717]: Same thing

File: website/templates/coach_dashboard.html [hunk 3]
```diff
@@ -507,7 +507,20 @@ <h4 class="card-title"> Athletes</h4>
                         <tbody>
                           <tr>
                             
-                            <td>
```
Comment (tymkiwdylan) [inline website/templates/coach_dashboard.html:510]: Where is athlete_data coming from? and all_users?

File: website/templates/peak_view.html [hunk 4]
```diff
@@ -115,7 +115,13 @@ <h6 class="p-3 mb-0">Messages</h6>
                     <div><a href=coach_dashboard.html>Soccer (M)</a></div>
                     <div><a href="coach_dashboard.html">Soccer(W)</a></div>
                     <div><a href="coach_dashboard.html">Track and Field</a></div>
-                    <div><a href="coach_dashboard.html">Cross Country</a></div>
+                    <div><a href="coach_dashboard.html">Cross Country</a></div> -->
+
+                    {% for team in teams %}
+                      <div><a href="coach_dashboard/{{ team.id }}">{{ team.name }}</a></div>
```
Comment (tymkiwdylan) [inline website/templates/peak_view.html:121]: This is good, but we need to change things in views.py for it to work

File: website/templates/peak_view.html [hunk 5]
```diff
@@ -205,24 +211,24 @@ <h6 class="card-title"> Calorie Intake </h6>
...
       </div>
-      <script src="{{url_for('static', filename = 'assets/vendors/js/vendor.bundle.base.js')}}"></script>
+      <script src="assets/vendors/js/vendor.bundle.base.js"></script>
     <!-- endinject -->
     <!-- Plugin js for this page -->
-    <script src="{{url_for('static', filename='assets/vendors/chart.js/Chart.min.js')}}"></script>
-    <script src="{{url_for('static', filename='assets/vendors/progressbar.js/progressbar.min.js')}}"></script>
-    <script src="{{url_for('static', filename='assets/vendors/jvectormap/jquery-jvectormap.min.js')}}"></script>
-    <script src="{{url_for('static', filename='assets/vendors/jvectormap/jquery-jvectormap-world-mill-en.js')}}"></script>
-    <script src="{{url_for('static', filename = 'assets/vendors/owl-carousel-2/owl.carousel.min.js')}}"></script>
```
Comment (tymkiwdylan) [inline website/templates/peak_view.html:215]: We need url_for

**Review by tymkiwdylan — CHANGES_REQUESTED** [review]
Try to solve the import assets issues and I can explain the backend for making the embedded python code work

File: website/templates/coach_dashboard.html (hunk 3 above)
Comment (isabella-feng) [reply website/templates/coach_dashboard.html:510]: placeholders for corresponding data (eg sleep, nutrition etc.) I don't know what to call them exactly hahah

Comment (isabella-feng) [reply website/templates/coach_dashboard.html:510]: and I wanted to get a list of all athletes in a team. not quite sure how to do it though

File: website/templates/peak_view.html (hunk 4 above)
Comment (isabella-feng) [reply website/templates/peak_view.html:121]: cool

File: website/templates/coach_dashboard.html (hunk 3 above)
Comment (tymkiwdylan) [reply website/templates/coach_dashboard.html:510]: Oh ok cool. I can show you how to do it, it's actually quite simple

**Pull request closed without merging**